python/ray/util/spark/databricks_hook.py

```diff
@@ -74,12 +83,18 @@ def _get_db_api_entry():
     return get_dbutils().entry_point
 
 
-_DATABRICKS_DEFAULT_TMP_DIR = "/local_disk0/tmp"
+@lru_cache(maxsize=1)
+def _get_start_hook():
+    if RAY_ON_SPARK_START_HOOK in os.environ:
+        return _load_class(os.environ[RAY_ON_SPARK_START_HOOK])()
+    if is_in_databricks_runtime():
+        return DefaultDatabricksRayOnSparkStartHook()
+    return RayOnSparkStartHook()
 
 
 class DefaultDatabricksRayOnSparkStartHook(RayOnSparkStartHook):
     def get_default_temp_dir(self):
```

Comment:
We should rename the method name to `get_default_temp_root_dir` to avoid confusion.